tests/unit/Makefile

```diff
@@ -23,9 +23,12 @@ test: ${ALL_TESTS}
 test_sanity: test_sanity.c
 test_x86: test_x86.c
 test_mem_map: test_mem_map.c
+${ALL_TESTS}: libunicorn.so.0
```

Comment:
have you considered that on OSX, the shared lib extension is `.dylib`, not `.so`?